### PR TITLE
Add initial implementation

### PR DESCRIPTION
**Pull request by klzns**
**What problem is this solving?**

Fix issue with Firefox blocking iframe by creating a native solution for Elfsight.

**How should this be manually tested?**

Open https://breno--storecomponents.myvtex.com/
Check the footer.

**Screenshots or example usage:**

Changed the theme blocks footer to:

```json
  "footer-layout.desktop": {
    "children": [
      "elfsight#foobar",
      "flex-layout.row#footer-1-desktop",
      "flex-layout.row#footer-2-desktop",
      "flex-layout.row#footer-3-desktop"
    ]
  },
  "elfsight#foobar": {
    "props": {
      "appId": "4a13d81d-20f3-4661-b472-9de777efe3ed"
    }
  },
```

image

### CODE REVIEW

**Review by kaisermann — APPROVED** [review]

File: manifest.json [hunk 1]
```diff
@@ -1,35 +1,15 @@
 {
-  "name": "YOURAPPNAME",
+  "name": "elfsight",
   "vendor": "vtex",
   "version": "0.0.1",
```
Comment (iaronaraujo) [inline manifest.json:4]:
```suggestion
  "version": "1.0.0",
```
:awesome:

File: docs/README.md [hunk 2]
```diff
@@ -53,4 +60,14 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
 <!-- DOCS-IGNORE:end -->
+
+---- 
+
+Check out some documentation models that are already live: 
+- [Breadcrumb](https://github.com/vtex-apps/breadcrumb)
+- [Image](https://vtex.io/docs/components/general/vtex.store-components/image)
+- [Condition Layout](https://vtex.io/docs/components/all/vtex.condition-layout@1.1.6/)
+- [Add To Cart Button](https://vtex.io/docs/components/content-blocks/vtex.add-to-cart-button@0.9.0/)
+- [Store Form](https://vtex.io/docs/components/all/vtex.store-form@0.3.4/)
```
Comment (iaronaraujo) [inline docs/README.md]: remove

**Review by iaronaraujo — CHANGES_REQUESTED** [review]

**Review by iaronaraujo — APPROVED** [review]

File: docs/README.md [hunk 3]
```diff
@@ -1,49 +1,61 @@
...
++  }
+```
 
-<!-- Remember to also **showcase any necessary disclaimer** related to the app in this section, such as the different behavior it may display during its configuration. -->
+### `elfsight` props
 
-## Modus Operandi *(not mandatory)*
+| Prop name    | Type            | Description    | Default value                                                                                                                               |
+| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
+| `appId`      | `string`       | The app id of the widget provided by the Elfsight in the format of: `XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX`.          | ``        |
```
Comment (guerreirobeatriz) [inline docs/README.md]:
```suggestion
| Prop name    | Type            | Description    | Default value                                                                                                                               |
| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
| `appId`      | `string`       | The app ID of the widget provided by Elfsight. The format should be as follows: `XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX`.      | `undefined`        |
```

**Review by pedrohenriqueoliveira — APPROVED** [review]

File: docs/README.md [hunk 4]
```diff
@@ -1,49 +1,61 @@
...
 
-There are scenarios in which an app can behave differently in a store, according to its configuration. It's crucial then to go through these **behavioral changes** in this section, allowing users to fully understand the **practical application** of the app in their store.
+You can get the app id from the Elfsight admin:
+
+![App Id](https://user-images.githubusercontent.com/284515/87574775-3d62a580-c6a5-11ea-8278-3090254b16af.png)
```
Comment (guerreirobeatriz) [inline docs/README.md]:
```suggestion
You should get the app ID from the Elfsight admin:

![App Id](https://user-images.githubusercontent.com/284515/87574775-3d62a580-c6a5-11ea-8278-3090254b16af.png)
```

File: docs/README.md [hunk 5]
```diff
@@ -1,49 +1,61 @@
...
 
-It is possible to install in your store either by using App Store or the VTEX IO Toolbelt.
+```json
+"dependencies": {
+  "vtex.elfsight": "1.x"
+}
+```
 
-### Using VTEX App Store
+2. Add the block `elfsight` to your theme `blocks.json` passing the prop `appId`.
```
Comment (guerreirobeatriz) [inline docs/README.md]: where should it be added to? Is there a specific page template?

**Review by guerreirobeatriz — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 5 above)
Comment (klzns) [reply docs/README.md]: It can be added anywhere

File: docs/README.md [hunk 6]
```diff
@@ -1,49 +1,61 @@
...
+# Elfsight
 
 <!-- DOCS-IGNORE:start -->
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 <!-- DOCS-IGNORE:end -->
 
-Under the app's name, you should explain the topic, giving a **brief description** of the **app's functionality** (what is it for?) in a store.
+Elfsight widgets help 300,000+ website owners to increase sales, engage visitors, collect leads and more.
```
Comment (guerreirobeatriz) [inline docs/README.md]:
```suggestion
Elfsight first party integration app. The [solution](https://elfsight.com/) provides widgets that help website owners to increase sales, engage visitors, collect leads and more.
```

File: docs/README.md (hunk 5 above)
Comment (guerreirobeatriz) [inline docs/README.md]:
```suggestion
2. Add the block `elfsight`, responsible for rendering your Elfsight widget, to your theme `blocks.json` file. Notice: the block can be added to any page template you want. 
3. Declare in it the prop `appId` whose value must be the app ID of the widget provided by Elfsight. For example:
```

File: docs/README.md [hunk 7]
```diff
@@ -1,49 +1,61 @@
...
+```
 
-### Using VTEX App Store
+2. Add the block `elfsight` to your theme `blocks.json` passing the prop `appId`.
 
-1. Access the **Apps** section in your account's admin page and look for the Icommkt box;
-2. Then, click on the **Install** button;
-3. You'll see a warning message about needing to enter the necessary configurations. Scroll down and type in your **NAME OF A SETTINGS FIELD** in the `NAME OF THE APP` field.
-4. Click on **Save**.
+Example:
```
Comment (guerreirobeatriz) [inline docs/README.md]:
```suggestion
```

**Review by guerreirobeatriz — CHANGES_REQUESTED** [review]

**Review by guerreirobeatriz — APPROVED** [review]

**Pull request merged**